try.html

```diff
@@ -24,13 +24,13 @@
       description: The latest web-based interactive development environment
```

Comment:
```suggestion
      description: The latest web-based interactive development environment. ⚠️experimental⚠️this uses the <a href="https://jupyterlite.readthedocs.io/en/latest/">JupyterLite</a> package, please report any unexpected behavior.
```

Reply:
How about including a link to Discourse (or a GitHub repo?) for reporting issues?

Reply:
+1 from me, I think that's a good idea to give people a call to actoin